securedrop/tests/test_crypto_util.py

```diff
@@ -19,6 +19,15 @@ def setUp(self):
     def tearDown(self):
         utils.env.teardown()
 
+    def test_word_lists_imports(self):
+        """Make sure word count for each list is correct and Guard
+        against regressions regarding not stripping file contents
+        correctly.
+        """
+        self.assertNotIn('',   crypto_util.words
```

Comment:
Fwiw PEP8 recommends line breaks before binary operators: https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator.